pkg/operator/controller/garden/components.go

```diff
@@ -693,3 +698,28 @@ func getAPIServerDomains(domains []string) []string {
 	}
 	return apiServerDomains
 }
+
+func (r *Reconciler) newNginxIngressController() (component.DeployWaiter, error) {
+	return sharedcomponent.NewNginxIngress(
+		r.RuntimeClientSet.Client(),
+		r.GardenNamespace,
+		r.GardenNamespace,
+		r.ImageVector,
+		r.RuntimeVersion,
+		map[string]string{
+			"enable-vts-status":            "false",
+			"server-name-hash-bucket-size": "256",
+			"use-proxy-protocol":           "false",
+			"worker-processes":             "2",
+			"allow-snippet-annotations":    "false",
+		},
+		nil,
```

Comment:
In the `Seed` controller we do this: https://github.com/gardener/gardener/pull/7945/files#diff-a3bc9d8bf428d542a1668556adcba0143538e5af8dad225753b26953049ceb18R1213

which basically reads the load balancer annotations from the `Seed` spec.

Don't we need the same here reading it from https://github.com/gardener/gardener/blob/85f5d9da6df9f35063018f5a8d3981a97f35c738/example/operator/20-garden.yaml#L28-L31